helm/templates/hypertrace-alert-engine/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.hypertraceAlertEngineConfig.enabled}}
```

Comment:
It's not only configMap, but deployment to be enabled/disabled. So, do you think it should be at the top level with var `hypertraceAlertEngine.enabled`

Reply:
makes sense, let me do that